string mission_id
string root_task
string[] task_label
string json_outcome
string[] resource
float64[] resource_state
geometry_msgs/Pose[] start
geometry_msgs/Pose[] end
string[] target		# if end point is moving target
string[] mass_class
string[] action_class
